actions/notification_actions.jsx

```diff
@@ -33,7 +33,9 @@ export function sendDesktopNotification(post, msgProps) {
     const userStatus = UserStore.getStatus(user.id);
     const member = ChannelStore.getMyMember(post.channel_id);
 
-    if (isChannelMuted(member) || userStatus === UserStatuses.DND) {
+    const dndEnabled = userStatus === UserStatuses.DND;
+    const oooEnabled = userStatus === UserStatuses.OUT_OF_OFFICE;
+    if (isChannelMuted(member) || dndEnabled || oooEnabled) {
```

Comment:
0/5 but since no other usage except for conditional block, I prefer direct comparison in the if statement instead of having const declaration.